src/components/TaskCard/AddTaskCard.jsx

```diff
@@ -2,9 +2,10 @@ import React, { useState } from "react";
 import PropTypes from "prop-types";
 import { sanitizeText } from "utils/helpers/sanitizeText.js";
 import { KEY_ENTER } from "utils/constant/form";
+import DeleteIcon from "components/Shared/Image/DeleteIcon";
 import "components/TaskCard/index.scss";
 
-const AddTaskCard = ({ onCreateTask }) => {
+const AddTaskCard = ({ onCreateTask, onCancelIconClick }) => {
```

Comment:
do not expose view specific/ internal state, prefer exposing the API by feature.

```suggestion
const AddTaskCard = ({ onCreateTask, onCancelTask }) => {
```